tests/component/dataset/domain/models/DatasetPreviewMother.ts

```diff
@@ -1,16 +1,52 @@
 import { faker } from '@faker-js/faker'
 import { DatasetPreview } from '../../../../../src/dataset/domain/models/DatasetPreview'
+import { DatasetCitationMother, DatasetLabelsMother, DatasetVersionMother } from './DatasetMother'
 
 export class DatasetPreviewMother {
   static createMany(count: number): DatasetPreview[] {
     return Array.from({ length: count }, () => this.create())
   }
 
   static create(props?: Partial<DatasetPreview>): DatasetPreview {
-    return {
+    const datasetPreview = {
       persistentId: faker.datatype.uuid(),
       title: faker.lorem.sentence(),
+      labels: DatasetLabelsMother.create(),
+      isDeaccessioned: false,
```

Comment:
It might be good to change isDeaccessioned from false to faker.datatype.boolean(), to get some deaccessioned datasets in the test data

Reply:
It wasn't that straightforward but I did the changes to show some deaccessioned datasets in the storybook

Reply:
looks great, thanks!